elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -98,24 +104,21 @@ button[aria-expanded="true"] {
     }
 
     // Adjust contents position on hover.
-    &:hover {
-      padding-left: calc(( #{pfe-var(container-spacer)} * 3 ) - 1px ) !important;
-      --pfe-accordion--BorderLeftWidth: #{pfe-var(surface--border-width--heavy)};
-
-      &::before {
-        left: calc( #{pfe-var(container-spacer)} * 1.3125 ); // 21px.
-      }
+    &:hover::before {
+      left: calc( #{pfe-var(container-spacer)} * 1.3125 ); // 21px.
```

Comment:
i'm not sure why this style was there in the first place but I dont think it was working before. I don't think we want to have the caret move on hover right?

I'm seeing this on the "Accordion component on dark",  "nested panel example", the nested accordion (disclosure) has a moving caret on hover
https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/

Reply:
Are you seeing that on master too?  I don't think I changed it so I'm hesitant to fix that here.

Reply:
Two possible use-cases: You could rename the component variables to add more specificity (nav to navigation or accordion to accordion-header) or you could copy the styles to a new component and want to update it there (without having to grep the entire stylesheet for updates). 🤷

Reply:
Nope, it doesn't happen in master (using the rename to main branch netlify link:
https://deploy-preview-935--happy-galileo-ea79c4.netlify.app//elements/pfe-accordion/demo/ )

So i think that variable was just not working in master previously?

Reply:
@castastrophe  was your comment about use cases intended for this thread: https://github.com/patternfly/patternfly-elements/pull/854#discussion_r452290816

Reply:
Removed it in the latest commit

Reply:
shoot, now the text overlaps the caret when the accordion panel is closed (on disclosures only)
https://deploy-preview-854--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/

![image](https://user-images.githubusercontent.com/456863/87442546-6d854800-c5c2-11ea-9fc6-ec5ecfa61cb7.png)

Reply:
I already pushed up the fix - do a refresh and let me know.

Reply:
aces! thank you :)